app/src/main/java/no/usn/mob3000/domain/repository/social/IFriendsRepository.kt

```diff
@@ -0,0 +1,15 @@
+package no.usn.mob3000.domain.repository.social
+
+import no.usn.mob3000.domain.model.auth.UserProfile
+import no.usn.mob3000.domain.model.social.FriendData
+/**
+ *
+ *  Interface for fetching friends and non-friends.
+ *
+ * @author Husseinabdulameer11
+ * @created : 2024-11-02

```

Comment:
KDoc property shouldn't have a colon here, it'll likely render weirdly. It should be <property> <value>.